Vegeting/Screens/GroupList/Component/ClubListCollectionViewCell.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  GroupListCell.swift
+//  Vegeting
+//
+//  Created by kelly on 2022/10/26.
+//
+
+import UIKit
+
+final class ClubListCollectionViewCell: UICollectionViewCell {
+    lazy var coverImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(systemName: "star")
+        return imageView
+    }()
+    
+    lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 2
+        label.tintColor = .black
+        return label
+    }()
```

Comment:
p1;

외부에서 접근하지 않는 부분은 private으로 변경 부탁드립니다~
```suggestion
     private lazy var coverImage: UIImageView = {
        let imageView = UIImageView()
        imageView.image = UIImage(systemName: "star")
        return imageView
    }()
    
     private lazy var titleLabel: UILabel = {
        let label = UILabel()
        label.numberOfLines = 2
        label.tintColor = .black
        return label
    }()
```